components/base/kinematicbase/ptgKinematics.go

```diff
@@ -18,12 +18,13 @@ import (
 	rdkutils "go.viam.com/rdk/utils"
 )
 
-var zeroInput = make([]referenceframe.Input, 3)
+var zeroInput = make([]referenceframe.Input, 4)
 
 const (
 	ptgIndex int = iota
 	trajectoryIndexWithinPTG
-	distanceAlongTrajectoryIndex
+	startDistanceAlongTrajectoryIndex
```

Comment:
can we please add simple explanations of what these start for

Reply:
:+1: